src/utils/api.ts

```diff
@@ -4,7 +4,7 @@ import { stringify } from 'qs';
 export const ITEMS_PER_PAGE = 10;
 const DEFAULT_MUSHAF = 4; // King Fahad Quran Complex 15 Lines Hafs text.
 
-const STAGING_API_HOST = 'https://staging.quran.com/api/qdc';
+const STAGING_API_HOST = 'https://api.quran.com/api/qdc';
```

Comment:
This is just for testing, will revert this back to staging when we're about to merge the PR

Reply:
Sounds good, let's revert this before merging iA.